requirements.txt

```diff
@@ -4,7 +4,7 @@ numpy
 pytest
 pyvisfile
 pymetis
-importlib-resources>=1.1.0; python_version < '3.9'
+importlib-resources
```

Comment:
This change strikes me as incorrect. Why is this right?

Reply:
Emirge can't parse this line and just does nonsense. Installing this package even for Py3.9 seems to have no ill effect (see e.g. https://pypi.org/project/importlib-resources/)

Reply:
At risk of adding unwanted noise here.. I think that this change might be OK simply because of this:
```
if sys.version_info < (3, 9):
    # importlib.resources either doesn't exist or lacks the files()
    # function, so use the PyPI version:
    import importlib_resources
else:
    # importlib.resources has files(), so use that:
    import importlib.resources as importlib_resources
```
This snippet seems to be OK even if both versions co-exist?

Reply:
Fair enough. Also, #244.